tools/cloud_functions/gcs_event_based_ingest/backfill.py

```diff
@@ -0,0 +1,175 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Command Line utility for backfilling gcs_ocn_bq_ingest cloud function
+"""
+import concurrent.futures
+import logging
+import os
+import pprint
+import sys
+from argparse import ArgumentParser, Namespace
```

Comment:
```suggestion
from argparse import ArgumentParser
from argparse import Namespace
```

Reply:
reasoning? this looks like java style suggestion on a python import. Many python communities will import several things from a single module in a single statement (under the hood it is slightly more performant IIUC).
Example in airflow even splitting on to multiple lines:
https://github.com/apache/airflow/blob/master/airflow/operators/check_operator.py#L23-L28

Reply:
tl;dr I'll refactor imports to just import full modules directly to be more explicit and change yapf to style google.


Alright I did some research:
It seems PEP-8 says this style fine ([link](https://www.python.org/dev/peps/pep-0008/#imports))
Interestingly Google style says always import entire module for namespacing reasons (https://google.github.io/styleguide/pyguide.html#22-imports)

For posterity I was totally wrong about any performance penalty of importing a whole module according to this [SO post](https://softwareengineering.stackexchange.com/questions/187403/import-module-vs-from-module-import-function)

Reply:
refactored imports to match google style recommendation in 4ae77a2